### DISCUSSION TITLE
Marketplace search ability to filter apps and actions

### DISCUSSION THREAD

**Discussion by gabriel-pineda** [community/community#127609] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new marketplace currently shows search results for both apps and actions.

The problem is when I'm searching for an app - I get flooded with a LOT of low quality actions.

Please add an ability to filter the search by app or action. 

Thanks

**Comment by ReenigneArcher** [comment]
Came to say the same...

Also, the star count it missing making it hard to judge quality/popularity.

**Comment by WesleyAtanasio** [comment]
I agree with you
